install.sh

```diff
@@ -4,12 +4,24 @@
 # Installer.
 #
 
+THIS_DIR="$(realpath "$(dirname "$0")")"
 INSTALL_ROOT=""
 MACHINE="nicole"
```

Comment:
??? How is this overridden for vesnin or vegman?

Reply:
Why do we need to override the source directory?